components/Header.tsx

```diff
@@ -144,6 +146,49 @@ export const Header = ({
                 {linkName}
               </Link>
             ))}
+            {drawerVisible ? (
+              <>
+                <Link
+                  href="https://www.simulationhockey.com"
+                  _hover={{ textDecoration: 'none' }}
+                  className={LINK_CLASSES}
```

Comment:
We should probably be passing the `isExternal` prop to these links since they direct the user out from the index. Also I'm not seeing the hover state for these items being matched with the items in the drawer.

Also we duplicate these items here and below so maybe we should store them in a separate object and iterate through them to add these items. That way if we add more we can do so easily and also we can make sure these links are kept in sync.

Probably something like:
```jsx
const moreLinks = [
  {
    name: "Forums"
    href: "https://www.simulationhockey.com"
  },
  {
    ...
  },
  ...
};

export const Header = (...) => {
  ...
  return (
    ...
    {moreLinks.map(({name, href}) => (
      <Link href={href} className="CLASSNAME HERE" ...>{name}</Link>
    )}
  ...

```
Also as an added benefit we can remove the LINK_CLASSES const since we only write them once!

Reply:
passing isExternal as target="_blank" as I couldnt get isExternal to work. Im sure you'll have more comments from the way i've implemented it. Didn't really understand what you were trying to convey with the code snippet you have